### PR TITLE
Add stricter formatters

### PR DESCRIPTION
**Pull request by jgiannuzzi**

### CODE REVIEW

**Review by suprjinx — DISMISSED** [review]

File: pkg/api/mlflow/controller/metric.go [hunk 1]
```diff
@@ -5,14 +5,15 @@ import (
...
 
-	"github.com/G-Research/fasttrackml/pkg/database"
 	"github.com/apache/arrow/go/v11/arrow"
 	"github.com/apache/arrow/go/v11/arrow/array"
 	"github.com/apache/arrow/go/v11/arrow/ipc"
 	"github.com/apache/arrow/go/v11/arrow/memory"
 	"github.com/gofiber/fiber/v2"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/G-Research/fasttrackml/pkg/database"
```
Comment (dsuhinin) [inline pkg/api/mlflow/controller/metric.go:15]: no needed empty line.

**Review by dsuhinin — APPROVED** [review]
just one comment.

File: pkg/api/mlflow/controller/metric.go (hunk 1 above)
Comment (jgiannuzzi) [reply pkg/api/mlflow/controller/metric.go:15]: well spotted!

Comment (jgiannuzzi) [reply pkg/api/mlflow/controller/metric.go]: addressed in https://github.com/G-Research/fasttrackml/pull/155/commits/95f95fda117dacbcd1f958d1e42d2f1f6749ed87

**Review by dsuhinin — APPROVED** [review]

**Pull request merged**